server/domains/service_shift.py

```diff
@@ -1,14 +1,14 @@
 """
-This module handles data converstion from dictionary to class obj or vice versa
+This module handles data conversion from 
+dictionary to class obj or vice versa.
 """
 import uuid
 import dataclasses
 
-
 @dataclasses.dataclass
-class Work:
+class ServiceShift:  # renamed ServiceShift from Work
```

Comment:
Let's remove the comment, because it will be confusing in the future. 
```suggestion
class ServiceShift:
```